app/controllers/accounts_controller.rb

```diff
@@ -12,6 +12,11 @@ def new
     @users = policy_scope(User)
   end
 
+  def edit
+    @users = policy_scope(User)
+    @account = authorize @project.accounts.find_by(id: params[:id])
```

Comment:
Ez a sor egész sokszor szerepel a mostani change-ek között. Légyszi szervezzük ki a `load_project` mintájára egy `load_account` függvénybe!

Nice to know: A `before_action` -öknek meg tudom adni, hogy csak bizonyos akciók esetén fussanak. pl.:
```ruby
class MyAwesomesController < ApplicationController
  before_action :load_my_awesome, only: [:show]
  
    def index
      # itt ugye nem lenne értelme a before_action-nek, mert valszeg hiányzik hozzá a params[:id]
    end
    
    def show
    end
    
    private
    
    def load_my_awesome
      @my_awesome = MyAwesome.find(params[:id]) # most nem írtam olyan szépen ahogy kérni szoktam, de érted
    end
end
```

Reply:
ez csak 2 helyen szerepel @-ossal ebben a formában, a destroy csak egy sima változóban használja és a create is máshogy használja, legyen így is külön?